src/Mutant/MutantExecutionResult.php

```diff
@@ -121,6 +123,11 @@ public function getMutantHash(): string
         return $this->mutantHash;
     }
 
+    public function getMutatorClass(): string
```

Comment:
is there a reason to use a public getter instead of making it a public property since it's readonly?

Reply:
it's for consistency - all other private readonly variables are exposed with public getters. If we need to use public readonly properties, this should be done everywhere in a separate PR